### DISCUSSION TITLE
Frequency rule

### DISCUSSION THREAD

**Discussion by yihuoyh** [jertel/elastalert2#608] [Q&A] [answered]
why？
![image](https://user-images.githubusercontent.com/95841184/145746734-4de6b079-f274-4696-80c6-95d06cdea708.png)

is it wrong?
![image](https://user-images.githubusercontent.com/95841184/145747128-769a4df6-8729-4ab6-a9c3-12f222a256fa.png)

**Comment by Singh-Shivani** [comment, marked as answer]
@yihuoyh  You can write anything you want to show in subject when elastalert gets triggered,  in the place of ``name: rule`` 
`name` basically equals to  `ElastAlert`(default value elastalert sends as per my understanding) and  after` : `you can write what you want in subject line. 
for example I want to get email alert and I want my subject to be `Elastalert sends this mail, please check`
then I'll write - 
`name : "Elastalert sends this mail, please check"`

In mails I'll get -
`ElastAlert : Elastalert sends this mail, please check`

**Reply by yihuoyh** [reply]
Thank you for your reply.
There will be an error when starting, the log is as follows 
![image](https://user-images.githubusercontent.com/95841184/145913921-5f3353f9-c8b1-4bd1-86c3-00bee4e2928e.png)

**Reply by Singh-Shivani** [reply]
@yihuoyh  write something else in the place of `rule` and then check.

**Reply by jertel** [reply]
You will need to paste your entire rule config, replacing any sensitive values/passwords, so that we can help.
